src/client/directory.ts

```diff
@@ -2,40 +2,63 @@
 /**
  * TypeScript definitions for objects returned by the Application Directory.
  *
- * These structures are defined by the App-Directory FDC3 working group. The definitions here are based on current
- * proposals and are not final. There may be minor differences between the current spec and the definitions here,
- * in order to support this demo.
+ * These structures are defined by the App-Directory FDC3 working group. The definitions here are based on the 1.0.0
+ * specification which can be found at https://fdc3.finos.org/appd-spec
  */
 
 /**
- * Type definition that is used wherever the API expects the name of an application to be passed.
+ * Type definition that is used wherever the API expects the ID of an application to be passed.
  *
  * It is not possible for TypeScript to verify that only valid application names are passed to the FDC3 API. This type
  * definition is more a hint to the callee, over a plain "string" argument.
  */
-export type AppIdentifier = string;
+export type DirectoryAppId = string;
+
+export interface DirectoryAppImage {
```

Comment:
Maybe go with `AppDir` as the prefix?

Maps a little closer to what the directory is named within FDC3, and a bit shorter.

Reply:
I agree with this, Directory- is a bit confusing for some of the interfaces/types.

Reply:
Discussed with Phil - just dropping the prefix entirely